### PR TITLE
Section

### PR DESCRIPTION
**Pull request by maksimgm**
@bengrunfeld

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,9 +29,290 @@
...
 
-	<section class="content">
+<section class="content">
+		<header>
+			<h1>NEWSLETTER</h1>
+		</header>
+		<article>
+			<div class="sign-up">
+				<h1>Sign-up</h1>
+					<p>The BBC newsletter is a comprehensive summary of the day's most important news, compiled by the world's largest and most trusted news organization, and delivered to your email inbox each morning. To subscribe, simply provide us with your email address and country. <a href="http://pages.email.bbc.com/samplenewsletter/">See a sample.</a></p>
```
Comment (bengrunfeld) [inline index.html]: spacing.

File: index.html [hunk 2]
```diff
@@ -29,9 +29,290 @@
...
+			</div>
+			<div class="form">
+				<div class="email">
+					<h2>Email Address</h2>
+						<form>
+							<input type="text" name="Email Address" placeholder="Enter your email" tabindex="1">
+						</form>
```
Comment (bengrunfeld) [inline index.html]: `form` is not a child of `h2`, so it should not be indented further than `h2`.

File: index.html [hunk 3]
```diff
@@ -29,9 +29,290 @@
...
+						<option value="VE">Venezuela, Bolivarian Republic of</option>
+						<option value="VN">Viet Nam</option>
+						<option value="VG">Virgin Islands, British</option>
+						<option value="VI">Virgin Islands, U.S.</option>
+						<option value="WF">Wallis and Futuna</option>
+						<option value="EH">Western Sahara</option>
+						<option value="YE">Yemen</option>
+						<option value="ZM">Zambia</option>
+						<option value="ZW">Zimbabwe</option>
+
```
Comment (bengrunfeld) [inline index.html]: remove whitespace.

File: index.html [hunk 4]
```diff
@@ -29,9 +29,290 @@
...
+					</select>
+
```
Comment (bengrunfeld) [inline index.html]: remove whitespace

File: style.css [hunk 5]
```diff
@@ -100,5 +99,95 @@ nav{
...
+
+input{
+	height: 40px;
+	width: 370px;
+	background: #e4e4e4;
+	padding-left: 15px;
+	font-weight: bold;
+	font-family: sans-serif;
+	font-size: 12px;
+}
```
Comment (bengrunfeld) [inline style.css:187]: you need to specify `border: none` to over-write the browser's inbuilt styles for `input`.

Comment (bengrunfeld) [inline style.css:187]: remember - by targeting `input`, you are affecting **every** input on the page. In a future job, you may or may not want to do that, depending on task you're given. I would definitely try to mitigate damage by giving the form a class (e.g. `class="newsletterForm"`) and then targeting the `input`s via that. I.e. `.newsletterForm input`.

**Comment by bengrunfeld** [comment]
Make a `/* Typography */` section where you define the font family, font size and font weight for all the text elements on your page.

Quiz Question: What is the font that BBC uses on their newsletter page?

**Comment by maksimgm** [comment]
BBC uses font-family san-serif.

What do you mean by a typography section?

Best,
Maksim

File: index.html [hunk 6]
```diff
@@ -29,9 +29,299 @@
...
-	<section class="content">
-
+<section class="content">
+	<header>
+		<h1>NEWSLETTER</h1>
+	</header>
+	<article>
+		<div class="sign-up">
+			<h1>Sign-up</h1>
+				<p>The BBC newsletter is a comprehensive summary of the day's most important news, compiled by the world's largest and most trusted news organization, and delivered to your email inbox each morning. To subscribe, simply provide us with your email address and country. <a href="http://pages.email.bbc.com/samplenewsletter/">See a sample.</a></p>
```
Comment (bengrunfeld) [inline index.html]: again, `p` is not a child of `h1`, so no extra indent needed

File: index.html [hunk 7]
```diff
@@ -28,10 +28,293 @@
...
+	</header>
+	<article>
+		<div class="sign-up">
+			<h1>Sign-up</h1>
+			<p>The BBC newsletter is a comprehensive summary of the day's most important news, compiled by the world's largest and most trusted news organization, and delivered to your email inbox each morning. To subscribe, simply provide us with your email address and country. <a href="http://pages.email.bbc.com/samplenewsletter/">See a sample.</a></p>
+		</div>
+		<div class="email">
+			<h2>Email Address</h2>
+		<form action="http://pages.email.bbc.com/success/" method="POST" class="email-form">
+		<input type="text" name="Email Address" placeholder="Enter your email" tabindex="1">
```
Comment (bengrunfeld) [inline index.html]: `input` is a child of form. please indent everything that is a child.

Comment (bengrunfeld) [inline index.html]: where is the closing `</form>` tag?

File: index.html [hunk 8]
```diff
@@ -28,10 +28,293 @@
...
+		</div>
```
Comment (bengrunfeld) [inline index.html]: Make sure you nest correctly. Check your above code and fix it up. E.g. The following is wrong

```
<div>
    <form>
</div>
    </form>
```

File: index.html [hunk 9]
```diff
@@ -28,10 +28,293 @@
...
+		<div class="radio-button">
+			<input type="radio" name="more-information" value="checkbox">
+			<p>Tick here if you wish to receive more information about BBC programming, products, services and offers.</p>
+		</div>	
+		<div class="subscribe-info">
+			<p>This information will be sent to you by BBC Global News Limited (operator of the BBC's international commercial news services) and BBC Worldwide Limited (operator of BBC's other commercial activities). You may opt out of receiving these from either company by clicking on the unsubscribe link in each company's emails.</p>
+		</div>
+		<div class="submit">
+			<input type="Submit" value="Submit">
+		</div>
```
Comment (bengrunfeld) [inline index.html]: this whole section is a child of `form`. as a child of `form`, it needs to be indented properly.

File: index.html [hunk 10]
```diff
@@ -28,10 +28,293 @@
...
+		<div class="subscribe-info">
+			<p>This information will be sent to you by BBC Global News Limited (operator of the BBC's international commercial news services) and BBC Worldwide Limited (operator of BBC's other commercial activities). You may opt out of receiving these from either company by clicking on the unsubscribe link in each company's emails.</p>
+		</div>
+		<div class="submit">
+			<input type="Submit" value="Submit">
+		</div>	
+		</form>	
+	</article>
+	<aside>
+			<div class="image">
```
Comment (bengrunfeld) [inline index.html]: only one index, not several. please check this same issue everywhere in your file.

Comment (maksimgm) [inline index.html]: What do you mean by one index?

**Comment by maksimgm** [comment]
I don't have any whitespace in my text editor. I went through the entire html and checked for whitespace.

File: index.html [hunk 11]
```diff
@@ -28,10 +28,293 @@
...
+			</div>
+			<div class="submit">
+				<input type="Submit" value="Submit">
+			</div>	
+		</form>	
+	</article>
+	<aside>
+			<div class="image">
+				<img src="envelope-img.png" alt="envelope-image"/>
+			</div>
```
Comment (bengrunfeld) [inline index.html]: spacing. should only have 1 indent, not several.

**Comment by bengrunfeld** [comment]
K, so a typography section looks something like this. Almost a CSS reset.

```
h1, h2, h3, h4, h5, h6, p, a {
  font-family: sans-serif;
  margin:0;
  padding:0;
}

```

**Pull request merged**